stdlib/Distributed/src/pmap.jl

```diff
@@ -152,10 +152,13 @@ function pmap(p::AbstractWorkerPool, f, c; distributed=true, batch_size=1, on_er
     end
 end
 
-pmap(p::AbstractWorkerPool, f, c1, c...; kwargs...) = pmap(p, a->f(a...), zip(c1, c...); kwargs...)
-pmap(f, c; kwargs...) = pmap(default_worker_pool(), f, c; kwargs...)
+pmap(f, p::AbstractWorkerPool, c1, c...; kwargs...) = pmap(a->f(a...), p, zip(c1, c...); kwargs...)
+pmap(f, c; kwargs...) = pmap(f, default_worker_pool(), c; kwargs...)
 pmap(f, c1, c...; kwargs...) = pmap(a->f(a...), zip(c1, c...); kwargs...)
 
+@deprecate pmap(p::AbstractWorkerPool, f, c; kwargs...) pmap(f, p, c; kwargs...)
+@deprecate pmap(p::AbstractWorkerPool, f, c1, c...; kwargs...) pmap(f, p, c1, c...; kwargs...)
```

Comment:
These should go in `base/deprecated.jl` near the bottom with a reference to the issue this closes. Other entries in that file should provide a decent example.

Reply:
I guess I was confused because this is an stdlib package. Do those deprecations still live in `base/deprecated.jl`?

Reply:
Oh you're absolutely right, sorry about that, I had forgotten that `pmap` moved to the stdlib. In that case it looks like other Distributed deprecations are done in `stdlib/Distributed/src/Distributed.jl` near the bottom.

Reply:
No worries! Just pushed another commit that moves them to `Distributed.jl`